Cannot find module 'bugger_ieee754' on debian

Hi- I'm not sure what info you need from me to help debug this- so I'll try and paste it all. If there's anything else you need, let me know!

````bash
path.existsSync is now called `fs.existsSync`.

module.js:340
    throw err;
          ^
Error: Cannot find module 'buffer_ieee754'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:280:25)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at encodeFloat (/srv/storage/live/hashtagstockexchange.com/collector/node_modules/hoard/lib/hoard.js:50:5)
    at Object.create (/srv/storage/live/hashtagstockexchange.com/collector/node_modules/hoard/lib/hoard.js:53:60)
    at Object.exports.ready (/srv/storage/live/hashtagstockexchange.com/collector/lib/writer.js:6:9)
    at Object.exports.start (/srv/storage/live/hashtagstockexchange.com/collector/lib/collector.js:24:10)
    at Object.<anonymous> (/srv/storage/live/hashtagstockexchange.com/collector/index.js:3:11)
    at Module._compile (module.js:456:26)
````

````bash
$ npm -version
1.3.11
````

````bash
$ lsb_release -a
No LSB modules are available.
Distributor ID:	Debian
Description:	Debian GNU/Linux 6.0.7 (squeeze)
Release:	6.0.7
Codename:	squeeze
````

````json
{
  "name": "hoard",
  "version": "0.1.5" ...
}
````

````javascript
var Hoard = require('hoard');

exports.ready = function ( callback ) {
  Hoard.create('htse.tweet', [[1,60],[10,600],[86400,10000]], function(err) {
    if (err) { throw err; }
    callback();
  });

}
````

--- a/src/hoard.coffee
+++ b/src/hoard.coffee
@@ -42,7 +42,7 @@ create = (filename, archives, xFilesFactor, cb) ->
         # Using 'buffer_ieee754' from node 0.5.x
         # as no libraries had a working IEEE754 encoder
         buffer = new Buffer(4)
-        require('buffer_ieee754').writeIEEE754(buffer, 0.5, 0, 'big', 23, 4);
+        require('./buffer_ieee754').writeIEEE754(buffer, 0.5, 0, 'big', 23, 4);
         buffer
 
     buffer = Put()

